ckanext/stcndm/validators.py

```diff
@@ -13,6 +13,31 @@
 import ckan.lib.navl.dictization_functions as df
 import ckanapi
 
+from ckanext.scheming.validation import (
+        scheming_validator, validators_from_string)
+from ckanext.fluent.validators import fluent_text_output
+
+LANG_SUFFIX = '_translated'
+
+@scheming_validator
+def fluent_core_translated_transitional_output(field, schema):
+    assert field['field_name'].endswith(LANG_SUFFIX), \
```

Comment:
ugh....the IDE did it like this I assume it was ok.

So much trouble for a temporary function.